app/containers/Avatar/interfaces.ts

```diff
@@ -9,7 +11,7 @@ export interface IAvatar {
 	size?: number;
 	borderRadius?: number;
 	type?: string;
-	children?: JSX.Element;
+	children?: React.ReactNode;
```

Comment:
We should avoid React.ReactNode.
I don't think there's a use case for it on React Native at all.

It let's you do this:
```tsx
<Avatar>
	anything
</Avatar>
```
That wouldn't be an issue on web, but it'd crash in RN.
For that to work on RN, we'd need to wrap the content with a Text element.
```tsx
<Avatar>
	<Text>anything</Text>
</Avatar>
```

```suggestion
	children?: React.ReactElement | null;
```

Good read: https://changelog.com/posts/the-react-reactnode-type-is-a-black-hole

Reply:
Nice, awesome point!